app/resources/parashot.json

```diff
@@ -0,0 +1,302 @@
+[
+   {
+      "name": "Parashat Vayechi",
+      "hebrew": "פרשת ויחי",
+      "link": "https://www.hebcal.com/sedrot/vayechi-20210102?i=on&utm_source=js&utm_medium=api",
+      "date": "2021-01-02"
```

Comment:
What will happen next year?

Reply:
Each year the Json file will need to be re-updated according to the API

Reply:
I think it will be nice to add an explaintion or documentation about that.

Reply:
I added an explanation to the comment in the function "get_parashot_from_json" from the file "load_parasha.py" :)